### ISSUE TITLE
Clean dependencies up

### ISSUE THREAD

**Issue by michwill** [zero-db/zerodb#24]
- repoze.catalog depends on ZODB3 but *actually* uses ZODB4 in zerodb. That needs to be fixed so that we don't download ZODB3
- lz4 compression should be moved into a separate plugin. For compression, we should use (optional) zlib level=2 by default

**Comment by micxjo** [comment]
If you're interested, I have a [fork](https://github.com/micxjo/repoze.catalog) of repoze.catalog that drops the ZODB3 dependency and adds python3.5 support (together with 2.6, 2.7 and pypy).

**Comment by michwill** [comment]
Huh, very interesting indeed!

**Comment by michwill** [comment]
@micxjo I wonder if it can be merged into original repoze.catalog repo. If not, we can keep it as a separate package in zerodbext namespace

**Comment by michwill** [comment]
Closing using work of @micxjo. Also python3 is supported!
